packages/turf-interpolate/index.d.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference types="geojson" />
+
+import {Points} from '@turf/helpers';
```

Comment:
Since `@turf/helpers` is not required in the main `package.json`, it might be better to include the Full GeoJSON typescript definition instead of importing it (in case the dependencies wouldn't include `@turf/helpers`).

```typescript
type Point = GeoJSON.FeatureCollection<GeoJSON.Point>;
```